extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/devui/KafkaDevUIProcessor.java

```diff
@@ -0,0 +1,55 @@
+package io.quarkus.kafka.client.deployment.devui;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.deployment.IsDevelopment;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.devui.spi.JsonRPCProvidersBuildItem;
+import io.quarkus.devui.spi.page.CardPageBuildItem;
+import io.quarkus.devui.spi.page.Page;
+import io.quarkus.kafka.client.runtime.devui.KafkaJsonRPCService;
+
+/**
+ * Kafka Dev UI (v2)
+ */
+public class KafkaDevUIProcessor {
+
+    private static final Logger log = Logger.getLogger(KafkaDevUIProcessor.class);
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    public CardPageBuildItem pages() {
+        CardPageBuildItem cardPageBuildItem = new CardPageBuildItem();
+
+        cardPageBuildItem.addPage(Page.webComponentPageBuilder()
+                .icon("font-awesome-solid:folder-tree")
+                .componentLink("qwc-kafka-topics.js")
+                .title("Topics"));
+        //  TODO: Implement this. This is also not implemented in the old Dev UI
```

Comment:
Yeah... schema registry is tricky as every schema registry has its own specific API.